src/SWP/Bundle/CoreBundle/Controller/ExternalOauthController.php

```diff
@@ -17,23 +18,25 @@ class ExternalOauthController extends Controller
     /**
      * @Route("/connect/oauth", name="connect_oauth_start")
      */
-    public function connectAction(): Response
+    public function connectAction(Request $request): Response
     {
+        $referer = $request->headers->get('referer');
+
         $clientRegistry = $this->get('knpu.oauth2.registry');
 
         return $clientRegistry
             ->getClient('external_oauth')
             ->redirect([
                 'openid', 'email', 'profile',
-            ]);
+            ], ["state" => $referer]);
```

Comment:
@kottkrig please use single quotes here as we use them everywhere so I can merge that PR. Besides that, the PR LGTM 👍

Reply:
Sure thing!